core/src/avm2.rs

```diff
@@ -102,7 +102,7 @@ impl<'gc> Avm2<'gc> {
             broadcast_list: HashMap::new(),
 
             #[cfg(feature = "avm_debug")]
-            debug_output: false,
```

Comment:
You accidentally committed your debug code

Reply:
It would be nice if there were a way to do this without editing the source code (since Ctrl-Alt-D doesn't work on my distro and is cumbersome to key in before any instruction gets run anyway)